src/backend/distributed/commands/statistics.c

```diff
@@ -506,6 +530,51 @@ GetExplicitStatisticsSchemaIdList(Oid relationId)
 }
 
 
+/*
+ * GetAlterIndexStatisticsCommands returns the list of ALTER INDEX .. ALTER COLUMN ..
+ * SET STATISTICS commands, based on non default targets of the index with given id.
+ * Note that this function only looks for expression indexes, since this command is
+ * valid for only expression indexes.
+ */
+static List *
+GetAlterIndexStatisticsCommands(Oid indexOid)
+{
+	List *alterIndexStatisticsCommandList = NIL;
+	int exprCount = 0;
+	while (true)
+	{
+		char *attName = GetAttNameWithExprCount(exprCount);
+
+		HeapTuple attTuple = SearchSysCacheAttName(indexOid, attName);
```

Comment:
Is there a reason to no use `SearchSysCacheAttNum` function? Looked like it could be useful.

Reply:
makes sense. so we won't have to use `GetAttNameWithExprCount` either. Can remove it